src/ui/components/filters/filteroptionscomponent.js

```diff
@@ -204,35 +204,33 @@ export default class FilterOptionsComponent extends Component {
     super.setState(Object.assign({}, data, {
       name: this.name.toLowerCase(),
       ...this.config,
-      showReset: this.config.showReset && selectedCount > 0,
+      displayReset: this.config.showReset && selectedCount > 0,
       expanded: this.expanded,
       selectedCount,
       isSingleOption: this.config.control === 'singleoption'
     }));
   }
 
   onMount () {
-    const selectedEls = DOM.queryAll(this._container, '.js-yxt-FilterOptions-checkboxInput:checked');
-    const selectedCount = selectedEls && selectedEls.length;
+    let selectedCount = this._getSelectedCount();
```

Comment:
Do we need this variable any more now that we call _getSelectedCount() in the click handler below?

Reply:
Ah yes true we do not, removing